.s2i/httpd-cfg/01-commercial.conf

```diff
@@ -579,6 +579,11 @@ AddType text/vtt                            vtt
     RewriteRule ^dedicated/support/summarizing-cluster-specifications.html /dedicated/osd_support/osd-summarizing-cluster-specifications.html [NE,R=301]
     RewriteRule ^dedicated/support/?(.*)$ /dedicated/osd_support/$1 [NE,R=301]
 
+    # Redirects for ZTP changes delivered in https://github.com/openshift/openshift-docs/pull/35889
+    RewriteRule ^scalability_and_performance/ztp-deploying-disconnected.html /scalability_and_performance/ztp-far-edge/ztp-deploying-far-edge-clusters-at-scale.html [NE,R=302]
```

Comment:
@bscott-rh Wouldn't these require `^(container-platform)/(4\.10|4\.11)/` part as well right ? I'm using an example value for versions.

Reply:
Ah you are right I think. Will update.

Reply:
What about 
```
   RewriteRule ^container-platform/(4\.10|4\.11)/scalability_and_performance/ztp-deploying-disconnected.html container-platform/$1/scalability_and_performance/ztp-far-edge/ztp-deploying-far-edge-clusters-at-scale.html [NE,R=302]
```

Reply:
Updated rules look good in the tester.

Reply:
Great. This PR will need to hang out until I get closer to merging the content update PR.